vignettes/transition_disruption_metric.Rmd

```diff
@@ -0,0 +1,552 @@
+---
+title: "Transition Disruption Metric"
+author: "Jackson Hoffart"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{transition_disruption_metric}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
```

Comment:
Instead `use_article()`. [Compare](https://github.com/2DegreesInvesting/r2dii.match/blob/main/vignettes/r2dii-match.Rmd).


This will help build the article with packages that are not declared as dependencies in DESCRIPTION. Right now CI is on fire:

```
Error: --- re-building ‘transition_disruption_metric.Rmd’ using rmarkdown
Quitting from lines 12-20 (transition_disruption_metric.Rmd) 
Error: Error: processing vignette 'transition_disruption_metric.Rmd' failed with diagnostics:
there is no package called 'ggplot2'
--- failed re-building ‘transition_disruption_metric.Rmd’
```

https://github.com/2DegreesInvesting/PACTA_analysis/runs/4117422753?check_suite_focus=true